conda/models/match_spec.py

```diff
@@ -238,7 +239,8 @@ def match(self, rec):
         return True
 
     def _match_individual(self, record, field_name, match_component):
-        val = getattr(record, field_name)
+        #val = getattr(record, field_name)
+        val = record.__dict__[field_name]
```

Comment:
Actually this one probably won’t work. All of the record fields are ultimately descriptors, and there can sometimes be a lot of dynamic translation depending on the field.

Reply:
yeah. that's the one that is breaking the CI.